package.json

```diff
@@ -36,14 +36,14 @@
   },
   "devDependencies": {
     "chai": "4.2.0",
-    "eslint": "6.8.0",
+    "eslint": "7.4.0",
     "esm": "3.2.25",
-    "mocha": "7.0.0",
-    "nyc": "15.0.0",
-    "rimraf": "3.0.2",
-    "rollup": "1.28.0",
-    "rollup-plugin-size": "0.2.1",
-    "rollup-plugin-terser": "5.1.3",
+    "mocha": "8.0.1",
+    "nyc": "15.1.0",
+    "rimraf": "^3.0.2",
```

Comment:
Pinned deps. ^3.0.2 vs 3.0.2

Reply:
It may have been less work to make that change yourself from GitHub 😉

How about we commit a package-lock.json if you're concerned about pinning dependencies?